src/frontends/tensorflow/src/op/concat.cpp

```diff
@@ -34,17 +34,47 @@ OutputVector translate_concat_op(const NodeContext& node) {
     get_const_input(node, axis_idx, &tf_concat_axis_vec);
```

Comment:
May I ask you please to add some description about difference between `Concat` and `ConcatV2`:
```
Note that axis is going to different input ports to Concat and ConcatV2: 0 - For Concat, N-1 - for ConcatV2
```

Reply:
Done